main.py

```diff
@@ -1,7 +1,41 @@
 import init_django_orm  # noqa: F401
 
+from db.models import Genre, Actor
 from django.db.models import QuerySet
 
+genres = [
+    "Western",
+    "Action",
+    "Dramma",
+]
+
+actors = [
+    ("George", "Klooney"),
+    ("Kianu", "Reaves"),
+    ("Scarlett", "Keegan"),
+    ("Will", "Smith"),
+    ("Jaden", "Smith"),
+    ("Scarlett", "Johansson"),
+]
```

Comment:
Since these variables are not used in places other than in the `main` function, you could define them inside of the `main` function